src/components/NavigationHeader.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import * as PropTypes from 'prop-types';
+import * as colors from 'kitsu/constants/colors';
+import { SimpleHeader } from './SimpleHeader';
+
+const styles = StyleSheet.create({
+  icon: {
```

Comment:
We often use this scheme for components

- ComponentNamedFolder
    - component.js
    - **styles.js**
    - index.js